spec/entities/configurations/kalibro_configuration_spec.rb

```diff
@@ -17,6 +17,14 @@
 require 'spec_helper'
 
 describe KalibroClient::Entities::Configurations::KalibroConfiguration do
+  describe 'accessors' do
+    it 'should have accessors for the needed attributes' do
```

Comment:
So that reinforces my opinion that we should use the code snippet I wrote.

Thanks for checking that it works, @dread-uo!